src/menu/menu.c

```diff
@@ -645,6 +717,9 @@ menu_process_cursor_motion(struct wlr_scene_node *node)
 	assert(node && node->data);
 	struct menuitem *item = node_menuitem_from_node(node);
 
+	if (!item || !item->selectable) {
+		return;
+	}
```

Comment:
Alternative: We could also use a new node descriptor for this instead of `item->selectable`. I think I'd prefer your current solution with the `selectable` bool though.